src/jobs/qa-automation-suite.yaml

```diff
@@ -0,0 +1,48 @@
+parameters:
+  test_path:
+    description: "The path to the test suite to run"
+    type: string
+  project_name:
+    description: "The name of the project in report portal"
+    type: string
+  launch_name:
+    description: "The name to give to the test launch"
+    type: string
+    default: EXTERNAL_TEST_RUN
+  launch_description:
+    description: "The description to give to the test launch"
+    type: string
+    default: "Automated Tests Ran by CircleCI"
+  launch_tag:
+    description: "The tags to add to the test launch (seperated by semi-colons)"
+    type: string
+    default: CIRCLECI;EXTERNAL
+  suite_version:
+    description: "The qa-automation-tests version to use"
+    type: string
+    default: "latest"
+
+executor:
+  name: qa-automation
+  qa_automation_runner_version: << parameters.suite_version >>
+
+environment:
+  TEST_PATH: << parameters.test_path >>
+  PROJECT_NAME: << parameters.project_name >>
+  LAUNCH_NAME: << parameters.launch_name >>
+  LAUNCH_DESCRIPTION: << parameters.launch_description >>
+  LAUNCH_TAG: << parameters.launch_tag >>
+  RP_ENDPOINT: https://reportportal.dev.codacy.org
```

Comment:
this URL should be a parameter, isn't it?

Reply:
The report portal URL is always the same, we only have one instance. Making this a parameter seems it will only increase the complexity of the parameters

Reply:
So I think I'm outdated ... we used to have an instance per environment (@jonathancarvalheiro  ?).
Since this code is public I don't know if we should keep this url hardcoded here, I think the best is to have it in env variable in the context.
We don't "publicize" our internal infrastructure urls.

Reply:
@andreaTP, no we always had one instance for all environments. 
But I agree as using that with an environment variable on the circle ci e.g.
There's some that we have to set on the circle ci in order for this to work:
RP_ENDPOINT; ENCRYPTION_KEY; RP_CIRCLECI_UUID
Could worth to be mentioned.